src/Kronecker.jl

```diff
@@ -2,10 +2,10 @@ module Kronecker
 
 export GeneralizedKroneckerProduct, AbstractKroneckerProduct, AbstractSquareKronecker
 export AbstractKroneckerSum, KroneckerSum
-export SquareKroneckerProduct, EigenKroneckerProduct, ShiftedKroneckerProduct
+export EigenKroneckerProduct, ShiftedKroneckerProduct
 export issquare, getmatrices, size, getindices, order, issymmetric, isposdef
 export ⊗, kronecker, Matrix, ⊕, kroneckersum
-export tr, det, logdet, collect, inv, +, *, mult!, eigen, /, adjoint, transpose, conj, solve, exp
+export tr, det, logdet, collect, inv, +, *, mult!, eigen, /, adjoint, transpose, conj, solve, exp, kron
 export getindex
```

Comment:
Also don't think that we need to export this since it's defined in `Base`.

Reply:
Not sure about this: should functions with a new method be exported?

Reply:
Nope.